Cargo.toml

```diff
@@ -8,23 +8,23 @@ clap = { version = "=4.4", features = ["cargo"] } # We force 4.4 so we don't hav
 serde = "1"
 serde_derive = "1"
 serde_json = "1"
-openssl = "0"
-base64 = "0"
+openssl = "0.10"
+base64 = "0.21"
```

Comment:
Accidental downgrade here, I think,

```
nix-shell-env ❯ cargo tree --depth 1 | grep base64
├── base64 v0.22.1
```

Reply:
we have different checkouts of that version, because it isn't pinned :/

Reply:
I've updated this crate anyways, but we can always do an actual update to latest dependencies after this gets merged.